examples/osgi-test-example-bnd/.gradle-wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
```

Comment:
We don't want gradlew in here. We did not put mvnw in the maven example. This just makes the example too large and confusing.

Reply:
removed gradle